packages/devtools/lib/src/inspector/inspector.css

```diff
@@ -15,24 +15,13 @@
 }
 
 .inspector-container {
-    height: 100%;
-    width: 100%;
-    display: flex;
     overflow: hidden;
     margin-top: 10px;
     align-items: center;
     flex: 1 1 auto;
-    flex-direction: column;
     box-sizing: border-box;
 }
 
-/* Minimum aspect ratio */
```

Comment:
why is this removed? This is needed to make the inspector switch between vertical and horizontal as the the window is resized.

Reply:
nm. was moved not removed.